pkg/jobcreator/controller.go

```diff
@@ -25,13 +26,18 @@ type JobCreatorController struct {
 	loop                  *system.ControlLoop
 	log                   *system.ServiceLogger
 	jobOfferSubscriptions []JobOfferSubscriber
+	moduleAllowlist       []string
 }
 
-// the background "even if we have not heard of an event" loop
-// i.e. things will not wait 10 seconds - the control loop
-// reacts to events in the system - this 10 second background
-// loop is just for in case we miss any events
-const CONTROL_LOOP_INTERVAL = 10 * time.Second
+const (
+	allowlistURL = "https://raw.githubusercontent.com/lilypad-tech/module-allowlist/main/allowlist.txt"
```

Comment:
[nit] Why is this one not in all caps? I may be off too, do let me know if there's some convention here.

Reply:
I updated `allowlistURL` to `ALLOWLIST_URL` to match the control loop const. I agree- better to match the convention and be neat!